modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -33,7 +33,7 @@ import org.osgi.service.component.annotations.Reference;
 /**
  * @author Nícolas Moura
  */
-public class UpgradeCatchAllCheck {
+public class UpgradeCatchAllCheck implements ConfigurationBeanDeclaration {
```

Comment:
@nicolas-sss, will this pattern only display a message, right?

If so, please forward this pull to Brian.